site/website/blog/2021-02-02-jet-44-is-released.md

```diff
@@ -0,0 +1,241 @@
+using logic that isn't sensitive to the exact event order. Jet can use
+this freedom to optimally load-balance the data across parallel tasks.
+This works great for stateless transforms like `map` and `filter` as
+well as aggregate operations specifically written in terms of
+commutative and associative functions. However, Jet also supports
+transforms such as `mapStateful`, where reordering any two events is
+likely to result in different output.
+
+In version 4.4 we provide a new option,
+`pipeline.setPreserveOrder(true)`, which tells Jet to disable the
+dataflow optimizations that result in reordered events. One consequence
+of enabling it is that the level of parallelism in the source stage
+determines the parallelism of all the subsequent stages because the data
+flows in parallel lanes through the pipeline. So if you have a source
+that isn't paralellized, your whole pipeline won't be parallelized
+either (at least until a stage that explicitly changes the order, such
+as `rebalance` or `groupingKey`). This feature works best when you have
+a partitioned source and you only require strict order among events with
+the same key. Then you get both the ordering you need and decent
+parallelization.
+
+## Full Release Notes
+
+Hazelcast Jet 4.4 is based on IMDG version 4.1.1. Check out its Release
+Notes [here](https://docs.hazelcast.org/docs/rn/index.html#4-1-1) and,
+for the Enterprise Edition,
+[here](https://docs.hazelcast.org/docs/ern/index.html#4-1-1).
+
+Members of the open source community that appear in these release notes:
+
+- @TomaszGaweda
+- @hhromic
+
+Thank you for your valuable contributions!
+
+### New Features
+
+- [sql] SQL Beta: submit jobs to Jet from the command-line SQL shell.
+  (#2595, #2636, #2648, #2654, #2665, #2729, #2763, #2788)
+
+- [file-api] [017] Unified API to create sources and sinks from
+  file-like resources: local filesystem, Amazon S3, Azure Blob Storage
+  and Data Lake Storage, Google Cloud Storage (#2518)
+
+- [kinesis] [018] Amazon Kinesis connector (#2656)
+
+- [pipeline-api] [016] Prevent event reordering: by default Jet reorders
+  data for performance, now you can disable this to get strict event
+  order where you need it.
+
+
+### Enhancements
+
+- [connectors] @hhromic improved the naming of source and sink stages
+  across different connectors, bringing them all in line with the same
+  convention `xSource` / `xSink` (#2685)
+
+- [pipeline-api] @TomaszGaweda added the `pipeline.isEmpty()` method
+  that tells whether it contains any stage (#2659)
+
+- [core] @TomaszGaweda added the
+  `jet.imdg.version.mismatch.check.disabled` config property that
+  disables the enforcement of the exact IMDG dependency version. This
+  allows adding IMDG quick fixes to the existing Jet release. (#2610)
+
+- [core] New packaging: download either the full package with all the
+  extensions enabled, or the minimal package and separately download the
+  extensions you want. (#2796)
```

Comment:
Correspondingly, there are now two Docker images - regular and slim.

Reply:
Addressed by adding a section in the main part of the blog post.